api/v1beta1/disruption_cron_webhook.go

```diff
@@ -92,8 +100,25 @@ func (d *DisruptionCron) ValidateDelete() (warnings admission.Warnings, err erro
 
 	log.Infow("validating deleted disruption cron", "spec", d.Spec)
 
+	// During the validation of the deletion the timestamp does not exist so we need to set it before emitting the event
+	d.DeletionTimestamp = &metav1.Time{Time: time.Now()}
```

Comment:
Okay this worries me a tiny bit. We don't need to do something similar in disruptions (as they live and send events for several minutes after being deleted), but the discrepancy here between the real timestamp and this one could confuse users.

We can make sure that any servers consuming the http events, are willing to take the latest deletion timestamp they get? That way if new events come later, that have the real deletionTimestamp of the object, those are used? wdyt?